packages/shared/src/db/queries.ts

```diff
@@ -350,6 +363,62 @@ export const insertGroups = createWriteQuery(
   ]
 );
 
+export const updateGroup = createWriteQuery(
```

Comment:
Optional: might be nice/pretty straightforward to add tests to some of these added queries, or at least any that might be error prone.